tck/features/SemanticErrorAcceptance.feature

```diff
@@ -204,7 +204,7 @@ Feature: SemanticErrorAcceptance
   Scenario: Failing when using MERGE on a node that is already bound
     When executing query:
       """
-      MATCH (a)
+      MERGE (a)
```

Comment:
Thanks for raising @swilly22 - this test is indeed erroneous, so well-spotted!
However, the test ought to be `MATCH (a) MERGE (a)`, as the test is checking that a failure occurs when attempting to merge a node that has already been found - via the match - and has thus been bound, meaning it is line 208 which is incorrect.

Reply:
Oh, sorry about that, I'll fix my PR.

Reply:
Super 👍